### DISCUSSION TITLE
Install fails with Access Denied

### DISCUSSION THREAD

**Discussion by mwadders** [iterate-ch/cyberduck#15141] [Q&A] [answered]
Trying to install version 8.3.6.40040, it fails with Access denied even though the installer is being run as admin.

![image](https://github.com/iterate-ch/cyberduck/assets/93320301/add53167-c4c5-4d82-bf46-c1c3f57123d0)

**Comment by AliveDevil** [comment, marked as answer]
Something (AV-software, some other application) modified the MSI Package Cache folder, and changed the permissions on that folder in a way, that prevents the Windows Installer from successfully running.

On my system, without ever touching that folder:
```
Get-Acl "C:\ProgramData\Package Cache" | FL


Path   : Microsoft.PowerShell.Core\FileSystem::C:\ProgramData\Package Cache
Owner  : BUILT IN\Administrators
Group  : WORKSTATION\None
Access : Everyone Allow  -1610612736
         Everyone Allow  ReadAndExecute, Synchronize
         NT-AUTHORITY\SYSTEM Allow  FullControl
         BUILT IN\Administrators Allow  FullControl
         BUILT IN\Users Allow  ReadAndExecute, Synchronize
         BUILT IN\Users Allow  -1610612736
Audit  :
Sddl   : O:BAG:S-1-5-21-3250444853-1395648116-134609186-513D:PAI(A;OICIIO;GXGR;;;WD)(A;;0x1200a9;;;WD)(A;OICI;FA;;;SY)(
         A;OICI;FA;;;BA)(A;;0x1200a9;;;BU)(A;OICIIO;GXGR;;;BU)
```

Please run the [Windows Installer troubleshooter](https://support.microsoft.com/topic/fix-problems-that-block-programs-from-being-installed-or-removed-cca7d1b6-65a9-3d98-426b-e9f927e1eb4d) in order to perform a check on all Windows Installer related paths.
